cli/azd/pkg/infra/provisioning/terraform/terraform_provider.go

```diff
@@ -76,7 +76,12 @@ func NewTerraformProvider(
 	return provider
 }
 
-func (t *TerraformProvider) Initialize(ctx context.Context, projectPath string, options provisioning.Options) error {
+func (t *TerraformProvider) Initialize(
+	ctx context.Context,
+	projectPath string,
+	options provisioning.Options,
+	_ osutil.ExpandableString,
```

Comment:
Wondering what if anything we should do here for Terraform, perhaps we should be adding this to `envVars` with something like `AZURE_RESOURCE_GROUP` so `tf` could observe it?